CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+
+## [1.1.0] - 2018-01-21
```

Comment:
As mentioned in the other PR, our changelog [conventions](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md#unreleased-change) all submitted changes go under `### [Unreleased]` there are a couple of reasons for this:
- There is no guarantee when a maintainer will review so the release date will likely be wrong unless we get to it in the same day
- There is no guarantee on the review/merge order, bumping the version prior to acceptance is pretty much guaranteed to need to have the submitter rebase
- Maintainer may disagree on how you interpret the change, for example in this case you versioned a as a patch when in fact it is a breaking change as you drop a version of ruby being supported. Even when an update is in the name of security we generally follow semver unless there is a very good reason not to such as putting the consumer at extreme risk. This is outlined as it relates to security [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_VERSION.md#security) if you are interested in understanding how we handle these edgecases.